src/creature.cpp

```diff
@@ -207,6 +220,11 @@ bool Creature::sees( const tripoint &t, bool is_player ) const
         if( has_effect( effect_no_sight ) ) {
             range = 1;
         }
+        if( range_mod > 0 ) {
+            if( range_mod < range ) {
+                range = range_mod;
```

Comment:
Inner check could be rewritten as `range = std::min( range, range_mod )`.

Reply:
Done, thanks.